### PR TITLE
gravatars

### PR DESCRIPTION
**Pull request by schacon**
CleanShot 2023-07-01 at 19 22 28@2x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,5 +97,8 @@
 		"xterm-addon-ligatures": "^0.6.0",
 		"xterm-addon-unicode11": "^0.5.0",
 		"xterm-addon-webgl": "^0.14.0"
+	},
+	"dependencies": {
+		"svelte-gravatar": "^1.0.3"
```
Comment (mtsgrd) [inline package.json]: I'm not sure why, but all other dependencies are added as devDependencies.. `pnpm install -D [package]`

File: src/routes/repo/[projectId]/Tray.svelte [hunk 2]
```diff
@@ -158,9 +159,15 @@
 					{#if branch.lastCommitTs > 0}
 						<div class="flex flex-row justify-between text-light-700 dark:text-dark-300">
 							<div class="text-sm">{formatDistanceToNow(branch.lastCommitTs * 1000)}</div>
-							<div title={branch.authors.join('\n')}>
+							<div
+								title={branch.authors.join('\n')}
+								class="isolate flex -space-x-1 overflow-hidden"
```
Comment (mtsgrd) [inline src/routes/repo/[projectId]/Tray.svelte:164]: `-space-x-1` has an incorrect `-` prefix

Comment (schacon) [reply src/routes/repo/[projectId]/Tray.svelte:164]: it's a negative spacing, so they overlap.

Comment (mtsgrd) [reply src/routes/repo/[projectId]/Tray.svelte:164]: TIL

**Pull request merged**